pkg/cloud/rgraph/exec/executor.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2023 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package exec
+
+import (
+	"context"
+
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud"
+)
+
+type ExecutorResult struct {
```

Comment:
mild go nit: this is package `exec`, so the `Executor` prefix may be redundant on some of the structs.  Any thoughts on `exec.UtorResult`? (I'm kidding, but `exec.ExecutorResult` does have the echo going on)

Reply:
fixed